gvsbuild/projects/libadwaita.py

```diff
@@ -0,0 +1,63 @@
+#  Copyright (C) 2016 - Yevgen Muntyan
+#  Copyright (C) 2016 - Ignacio Casal Quinteiro
+#  Copyright (C) 2016 - Arnavion
+#
+#  This program is free software; you can redistribute it and/or modify
```

Comment:
you added the copyright twice, also add your name

Reply:
Okayy!

Reply:
OMGG Why did I accidentally put it twiceee 😭😭😭